neural_compressor/torch/quantization/layers.py

```diff
@@ -149,7 +149,7 @@ def __init__(
         in_features,
         out_features,
         bits,
-        groupsize,
+        group_size,
```

Comment:
For the `layers.py`, suggests following the [stock PyTorch folder structure](https://github.com/pytorch/pytorch/tree/main/torch) and putting it at `neural_compressor/torch/nn`.

Reply:
The whole path is `torch/nn/modules/linear.py`, I think `modules.py` it's better. That's a history issue.
Before I named it as `modules.py` while you changed it to layers to avoid the same name usage in `habana_fp8` folder.
As the `habana_fp8` folder I will change it to modules again.

Reply:
Got it, thanks.